.travis.yml

```diff
@@ -43,6 +43,6 @@ script:
   - $ISLANDORA_DIR/tests/scripts/line_endings.sh sites/all/modules/islandora_solution_pack_newspaper
   - drush coder-review --reviews=production,security,style,i18n,potx,sniffer islandora_newspaper
   - phpcpd --names *.module,*.inc,*.test sites/all/modules/islandora_solution_pack_newspaper
-  - drush test-run --uri=http://localhost:8081 "Islandora Newspaper"
+  - php scripts/run-tests.sh --php `phpenv which php` --url http://localhost:8081 --verbose --class IslandoraNewspaperPageIngestTestCase
```

Comment:
Should probably make a ticket for this change, and track where its done, so we remember to do it everywhere eventually. Maybe with a little bit of documentation about the troubles with travis and drush removing the testing function, so other people can change other modules.